src/utils/__tests__/ranges.test.ts

```diff
@@ -329,9 +328,9 @@ describe("utils - ranges", () => {
         [20, 30],
         [50, 70],
       ]);
-      expect(getPlayedSizeOfRange(timeRanges, 0)).to.equal(0 - 0);
-      expect(getPlayedSizeOfRange(timeRanges, 9.9)).to.equal(9.9 - 0);
-      expect(getPlayedSizeOfRange(timeRanges, 54.6)).to.equal(54.6 - 50);
+      expect(getPlayedSizeOfRange(timeRanges, 0)).toBe(0 - 0);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Correct one of the identical sub-expressions on both sides of operator "-" ([typescript:S1764](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS1764))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=439&issues=AWgpECov_MPaBDn0IVTv&open=AWgpECov_MPaBDn0IVTv)

[ISSUE_KEY]: AWgpECov_MPaBDn0IVTv